_posts/2019-05-06-oled.md

```diff
@@ -159,8 +171,12 @@ And then the last extra piece you need is to select the MONO theme after initial
 ```c
     oledInit();
     
+    static lv_disp_buf_t disp_buf;    /*Descriptor of a display buffer */
+    static lv_disp_drv_t disp_drv;    /*Descriptor of a display driver*/
+    static lv_color_t gbuf[BUFSIZE];  /*Memory area used as display buffer */
+    
     lv_init();
-    lv_disp_drv_t disp_drv;                         /*Descriptor of a display driver*/
+    lv_disp_buf_init(&disp_buf, gbuf, NULL, BUFSIZE); /* BUFSIZE can be smaller than the actual display */
```

Comment:
`BUFSIZE` is a bit dangerous. People are likely to not read part of this tutorial and just copy the code, and when they get a message that `BUFSIZE` is not declared, they'll look it up and find that POSIX has a `BUFSIZ` variable.

Possibly it should be named `DISP_BUF_SIZE` instead?

Reply:
Agreed, I renamed it and added an extra explanation for good measure.